nf_core/components/components_utils.py

```diff
@@ -279,3 +280,10 @@ def _iterate_input_output(type) -> DictWithStrAndTuple:
     # If the tool name was not found in the response
     log.warning(f"Could not find an EDAM ontology term for '{tool_name}'")
     return None
+
+
+def try_generate_container_configs(directory: str | Path, path: str):
```

Comment:
```suggestion
def try_generate_container_configs(directory: Path, path: Path):
```
let's make this simpler